backend/database/factories/UserFactory.php

```diff
@@ -16,7 +16,7 @@ public function definition() {
             'name' => $this->faker->name(),
             'email' => $this->faker->unique()->safeEmail(),
             'email_verified_at' => now(),
-            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
+            'password' => 'password',
```

Comment:
use `Hash` to decrypt the hash, this changes you made is a `string password`. 
Related topic: https://laravel.com/docs/7.x/hashing

Reply:
resolved